client/reader/feed-stream/index.jsx

```diff
@@ -1,61 +1,77 @@
-var React = require( 'react' ),
-	url = require( 'url' );
-
-var EmptyContent = require( './empty' ),
-	DocumentHead = require( 'components/data/document-head' ),
-	Stream = require( 'reader/stream' ),
-	FeedHeader = require( 'reader/feed-header' ),
-	FeedStore = require( 'lib/feed-store' ),
-	FeedStoreActions = require( 'lib/feed-store/actions' ),
-	FeedStoreState = require( 'lib/feed-store/constants' ).state,
-	FeedError = require( 'reader/feed-error' ),
-	HeaderBack = require( 'reader/header-back' ),
-	SiteStore = require( 'lib/reader-site-store' ),
-	SiteState = require( 'lib/reader-site-store/constants' ).state;
-
-var FeedStream = React.createClass( {
-
-	getDefaultProps: function() {
-		return { showBack: true };
-	},
-
-	getInitialState: function() {
-		var feed = this.getFeed(),
-			site = this.getSite( feed ),
-			title = this.getTitle( feed, site );
-
-		return {
-			title,
-			feed,
-			site
-		};
-	},
+/**
+ * External dependencies
+ */
+import React from 'react';
+import url from 'url';
+import localize from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import EmptyContent from './empty';
+import DocumentHead from 'components/data/document-head';
+import Stream from 'reader/stream';
+import FeedHeader from 'reader/feed-header';
+import FeedStore from 'lib/feed-store';
+import FeedStoreActions from 'lib/feed-store/actions';
+import { state as FeedStoreState } from 'lib/feed-store/constants';
+import FeedError from 'reader/feed-error';
+import HeaderBack from 'reader/header-back';
+import SiteStore from 'lib/reader-site-store';
+import { state as SiteState } from 'lib/reader-site-store/constants';
+
+class FeedStream extends React.Component {
+
+	static propTypes = {
+		feedId: React.PropTypes.number.isRequired
+	};
+
+	static defaultProps = {
+		showBack: true
+	};
+
+	constructor( props ) {
```

Comment:
ohhhhh, yeah. since `getState` also uses the property initializer syntax, it has to be defined before use. Moving `getState` above the assignment statement should make it work.